lib/src/line_splitting/solve_state_queue.dart

```diff
@@ -14,7 +14,9 @@ import 'solve_state.dart';
 /// overflow characters. When a new state is added to the heap, it will be
 /// discarded, or a previously enqueued one will be discarded, if two overlap.
 class SolveStateQueue {
-  /// Initial capacity of a queue when created, or when added to after a [clear].
+  /// Initial capacity of a queue when created, or when added to after a
+  /// `clear`.
```

Comment:
Zero idea where "clear" went.

Reply:
This class was copied from HeapPriorityQueue and then heavily modified, so this comment is just referring to something that no longer applies to this class. You can just remove everything after "created".

Reply:
done